fili-core/src/main/java/com/yahoo/bard/webservice/web/TableFullViewProcessor.java

```diff
@@ -95,14 +98,44 @@ public TableGrainView formatTableGrain(LogicalTable logicalTable, String grain,
         resultRow.put("longName", StringUtils.capitalize(grain));
         resultRow.put("description", "The " + logicalTable.getName() + " " + grain + " grain");
         resultRow.put("retention", logicalTable.getRetention().toString());
-        resultRow.put(
-                "dimensions",
-                DimensionsServlet.getDimensionListSummaryView(logicalTable.getDimensions(), uriInfo)
-        );
+        resultRow.put("dimensions", getDimensionListFullView(logicalTable.getDimensions(), uriInfo));
```

Comment:
What is the reason we make this change here instead of in `DimensionsServlet`? If we make change in `DimensionsServlet` things are in sync which creates less duplication and complication that feels nicer unless there is a reason for not doing so.